Enhanced Documentation, Fix "stop after x messages"

SimonButtazzi opened the pull request:
Added some words about configuration and output examples

SimonButtazzi commented:
Fixed "stop after" x messages (pre_stop state).
State didn't work for stop after > 1.

xoseperez commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
     "name": "node-red-contrib-power-monitor",
-    "version": "1.2.0",
+    "version": "1.2.3",
```
Set it to 1.2.1

SimonButtazzi replied:
Hey, can you please merge?
Fix was committed, see last comment.

Thanks, Simon

xoseperez reviewed (changes requested):
Since this is a minor change and bug fix, version should ber 1.2.1

SimonButtazzi commented:
> Since this is a minor change and bug fix, version should be 1.2.1
You are right, initially I added docs and fixed the bug in a new version.
-> Changed version

xoseperez commented:
Merged and updated on npmjs.
